data_generation/FAO.py

```diff
@@ -0,0 +1,618 @@
+"""Run this via `make climate_categories/data/FAO.yaml` in the main
+directory."""
+
+import pathlib
+
+import climate_categories as cc
+
+OUTPATH = pathlib.Path("./climate_categories/data/FAO.yaml")
+
+
+def main():
+    spec = {
+        "name": "FAO",
+        "title": (
+            "Food and Agriculture Organization of the United Nations (FAO) "
+            "FAOSTAT data set categorisation"
+        ),
+        "comment": "Needed to add FAOSTAT data to PRIMAP-hist",
+        "references": "",
+        "institution": "FAO",
+        "hierarchical": True,
+        "last_update": "2024-12-10",
+        "version": "2024",
+        "total_sum": True,
+        "canonical_top_level_category": "0",
+    }
+
+    categories = {}
+
+    # 0. main categories
+    categories["0"] = {
+        "title": "Total",
+        "comment": "All emissions and removals",
+        "children": [["1", "2", "3", "4", "5", "6", "7"]],
+    }
+    children_1 = ["1.A", "1.B"]
+    children_2 = ["2.A", "2.B", "2.C", "2.D", "2.E"]
+    children_3 = [f"3.{i}" for i in "ABCDEFGHIJKLMNOPQR"]
+
+    main_categories = (
+        # category code, name and comment, gases, children
+        ("1", "Crops", ["CH4", "N2O"], children_1),
+        (
+            "2",
+            "Energy use in agriculture",
+            ["CH4", "N2O", "CO2"],
+            children_2,
+        ),
+        ("3", "Livestock", ["CH4", "N2O"], children_3),
+        # ("4", "Forest", ["CO2"], children_4),
```

Comment:
Maybe add a comment explaining why this line and the following are commented out?